packages/shopify-codemod/test/fixtures/mocha-context-to-global-references/rename.output.js

```diff
@@ -0,0 +1,17 @@
+suite('test', function() {
+  afterEach(function() {
+    sinon.stub(foo, 'bar');
+  });
+
+  before(function() {
+    this.testSpy = sinon.spy();
```

Comment:
That would be up the user of the codemod; for us, we'll make sure that the global we change things to is the same sandbox being injected into the test context.